recipes/graphvite/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set version = "0.2.2" %}
+
+package:
+  name: graphvite
+  version: {{ version }}
+
+source:
+  url: https://github.com/DeepGraphLearning/graphvite/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 81c0a55102051d2933d5b51dbf49e0b6faeefb2f503331b59484134e4db4586f
+  patches:
+    - patches/0001-allow-setting-CUDA_ARCHES-through-cmake-itself.patch
+    - patches/0002-compile-with-c-17.patch
+    - patches/0003-point-to-correct-faiss-lib.patch
+    - patches/0004-add-install-step.patch
+
+build:
+  number: 0
+  # only linux for now
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version in ("None", "10.2")]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+outputs:
+  # only difference between graphvite-mini and graphvite is the amout on runtime-deps
+  - name: graphvite-mini
```

Comment:
We have a convention that reduced dependency packages are [appended with the word "base"](https://anaconda.org/search?q=base), so your package would be "graphvite-base". Is that something you could switch to without disrupting packages that depend on your package?

Reply:
This reflects upstream nomenclature, which I want to reflect.